src/gutter-component.coffee

```diff
@@ -3,6 +3,7 @@ React = require 'react-atom-fork'
 {div} = require 'reactionary-atom-fork'
 {isEqual, isEqualForProperties, multiplyString, toArray} = require 'underscore-plus'
 Decoration = require './decoration'
+{$} = require './space-pen-extensions'
```

Comment:
We're trying to remove jQuery dependencies wherever we can, this probably won't get merged with this require.